src/components/home-cards/vacations-card.tsx

```diff
@@ -0,0 +1,22 @@
+import { Grid, Card, CardContent } from "@mui/material";
+import strings from "../../localization/strings";
+import { List } from "@mui/icons-material";
+import { Link } from "react-router-dom";
+
+const VacationsCard = () => (
+  <Card>
+    <CardContent>
+      <h3 style={{ marginTop: 6 }}>{strings.tableToolbar.myRequests}</h3>
+      <Grid container>
+        <Grid item xs={1}>
+          <List />
+        </Grid>
+        <Grid item xs={11}>
+          <Link to={"/vacations"}>Manage my vacation requests</Link>
```

Comment:
Perhaps this could just be vacations?
Also create a future trello task for designing and implementing the functionality for the vacations card, perhaps a string listing the next upcoming vacation dates and remaining vacation days. This should be done in a separate feature through

Reply:
Agreed. Made a new Trello task.